pkg/broker/storage.go

```diff
@@ -268,7 +269,24 @@ begin
+		end if;
+
+		if (select count(*) from plans where plan = '0b74a5e8-ad17-4209-bc61-a08520c3cc1f') = 0 then
+			insert into plans
+				(plan, service, name, human_name, description, version, type, scheme, categories, cost_cents, preprovision, attributes, provider, provider_private_details)
+			values
+				('0b74a5e8-ad17-4209-bc61-a08520c3cc1f', '01bb60d2-f2bb-64c0-4c8b-ead731a690bd', 'premium-t3-0',  'Premium-T3-0 (10.4)', 'Postgres 10.4 - 2xCPU 2GB Ram 20GB Storage (T3 Instance)', '10.4', 'postgres', 'postgres', 'Data Stores', 2000,  0, '{"compliance":"", "supports_extensions":true,  "ram":"2GB",   "database_replicas":true,  "database_logs":true,  "restartable":true,  "row_limits":null, "storage_capacity":"20GB",  "data_clips":true,  "connection_limit":null, "high_availability":false,  "rollback":"14 days", "encryption_at_rest":true, "high_speed_ssd":true,  "burstable_performance":false, "dedicated":true  }', 'aws-instance',    '{"AllocatedStorage":20,"AutoMinorVersionUpgrade":true,"AvailabilityZone":null,"BackupRetentionPeriod":14,"CharacterSetName":null,"CopyTagsToSnapshot":true,"DBClusterIdentifier":null,"DBInstanceClass":"db.t3.small","DBInstanceIdentifier":null,"DBName":null,"DBParameterGroupName":null,"DBSecurityGroups":null,"DBSubnetGroupName":"rds-postgres-subnet-group","Domain":null,"DomainIAMRoleName":null,"EnableCloudwatchLogsExports":null,"EnableIAMDatabaseAuthentication":null,"EnablePerformanceInsights":false,"Engine":"postgres","EngineVersion":"10.4","Iops":null,"KmsKeyId":null,"LicenseModel":null,"MasterUserPassword":null,"MasterUsername":null,"MonitoringInterval":null,"MonitoringRoleArn":null,"MultiAZ":false,"OptionGroupName":null,"PerformanceInsightsKMSKeyId":null,"PerformanceInsightsRetentionPeriod":null,"Port":null,"PreferredBackupWindow":null,"PreferredMaintenanceWindow":null,"ProcessorFeatures":null,"PromotionTier":null,"PubliclyAccessible":false,"StorageEncrypted":true,"StorageType":"gp2","Tags":null,"TdeCredentialArn":null,"TdeCredentialPassword":null,"Timezone":null,"VpcSecurityGroupIds":null}');
```

Comment:
can we name this ... I dunno premium-nano-0?... I realize its all just arbitrary I just don't want us to align ourselves too closely to a cloud provider with our nomenclature and naming. I guess its a nitpick so feel free to ignore me.

Reply:
The only reason I called it "premium-t3" is because the T3 instances behave differently with CPU bursting vs the old T2 instances and I wasn't sure if we wanted to make this clear to users. I'm fine renaming it if we wanna be agnostic